core/src/main/scala/com/banno/kafka/TopicPartitionOffset.scala

```diff
@@ -1,26 +0,0 @@
-/*
```

Comment:
I think I'm missing context. Why did this one get dropped?

Reply:
It's subsumed by https://github.com/Banno/kafka4s/pull/425/files#diff-6d08a03f2b6664bedea88f00caebbc2f2ace923dc3371a5e8dba0fb9b425e89bR11. It's a breaking change, but we're on milestones, and we make no guarantees. For the time being, I turned the implicit `toMap` conversion into an explicit method.

Reply:
Got it. I had a vague memory of something like that but couldn't remember what file it was in.